molo/core/templatetags/core_tags.py

```diff
@@ -352,22 +367,19 @@ def get_next_tag(context, tag):
     current_tag = tag.get_main_language_page()
     qs = Tag.objects.descendant_of(request.site.root_page).filter(
         languages__language__is_main_language=True).live()
-    if qs:
+    if qs.exists():
         tags = list(qs)
-        if len(tags) > 1:
```

Comment:
The `.exists()` makes this if statement redundant. Removed and block below shift indentation to the left.